x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/components/header_actions/search_playground_popover.tsx

```diff
@@ -23,15 +24,16 @@ export const SearchPlaygroundPopover: React.FC<SearchPlaygroundPopoverProps> = (
   indexName,
   ingestionMethod,
 }) => {
-  const playgroundUrl = PLAYGROUND_PATH + `?default-index=${indexName}`;
+  const playgroundUrl = APPLICATIONS_PLUGIN.URL + PLAYGROUND_PATH + `?default-index=${indexName}`;
```

Comment:
**non-blocking nitpick:** 

Since we're already using a template string here, making this a single template would be better than using `+` to concatenate IMO.
 
```suggestion
  const playgroundUrl =  `${APPLICATIONS_PLUGIN.URL}${PLAYGROUND_PATH}?default-index=${indexName}`;
```